var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -66,17 +67,20 @@ class PyAwkward(PythonPackage):
         ("@2.0.8", "@9"),
         ("@2.0.9", "@10"),
         ("@2.0.10", "@11"),
-        ("@2.1.0:", "@12"),
+        ("@2.1.0", "@12"),
```

Comment:
```suggestion
        ("@2.1.0:2.1.1", "@12"),
```
otherwise `py-awkward@2.1.1` won't depend on any version of `py-awkward-cpp`.